pkgs/development/dotnet-modules/python-language-server/default.nix

```diff
@@ -25,6 +26,8 @@ buildDotnetModule rec {
   dotnet-sdk = dotnetCorePackages.sdk_3_1;
   dotnet-runtime = dotnetCorePackages.runtime_3_1;
 
+  dotnetRestoreFlags = [ "--runtime ${dotnet-sdk.systemToDotnetRid stdenvNoCC.targetPlatform.system}" ];
```

Comment:
Maybe we should consider adding this flag by default to dotnet-restore-hook.sh?

If yes, then we should also consider adding an ability to override it (when ie. building a windows version)

Reply:
I actually did try that locally: https://github.com/corngood/nixpkgs/commit/204c2998506152d0c1d4a744ad1f6dbf51233c33

It breaks a bunch of packages: https://gist.github.com/corngood/902393ad085e7b2d447e52b877dbfde3

I still think it's a good idea, but I'm not sure it can be done universally due to all the hacky stuff projects do with properties.  For example:

> /nix/store/byipzqmp760qz1nj0cncvjln1zpdd6xa-dotnet-sdk-6.0.401/sdk/6.0.401/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(90,5): error NETSDK1083: The specified RuntimeIdentifier 'linux-x64;linux-x64' is not recognized. [/build/source/osu.Game.Rulesets.Catch/osu.Game.Rulesets.Catch.csproj]

Reply:
Yeah, the cli is a bit buggy. We can make an issue for it and make the change in the future

Reply:
Since it is such a common flag it would maybe be nice to have an `dotnetUseRid` option so that `buildDotnetModule` can set it for you.